tonic/src/extensions.rs

```diff
@@ -71,3 +71,26 @@ impl fmt::Debug for Extensions {
         f.debug_struct("Extensions").finish()
     }
 }
+
+/// A gRPC Method info extension.
+#[derive(Debug)]
+pub struct GrpcMethod {
+    service: &'static str,
+    method: &'static str,
+}
+
+impl GrpcMethod {
+    /// Create a new `GrpcMethod` extension.
+    pub fn new(service: &'static str, method: &'static str) -> Self {
```

Comment:
I would like to make this `#[doc(hidden)]` so that codegen can use it but users can't find it in the docs since I am not sure if we want to expose the fact that internally its `&'static`.